README.md

```diff
@@ -60,7 +60,7 @@ cfn test --log-group-name cw_log_group --log-role-arn log_delivery_role_arn # Ha
 ```
 
 Note:
-* To use your type configuration in contract tests, you will need to save your type configuration json file in `~/.cfn-cli/typeConfiguration.json`.
+* To use your type configuration in contract tests, you will need to save your type configuration json file in `~/.cfn-cli/typeConfiguration.json` or specify the file with `--typeconfig <alternate file including location>`
```

Comment:
We should be more specific about the format of the argument. Is it an absolute path?

Reply:
Let me add in more details